ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentBaselines.tsx

```diff
@@ -1,14 +1,30 @@
 import React from 'react';
-import { Flex, FlexItem, Stack, StackItem, Switch, Tooltip } from '@patternfly/react-core';
+import {
+    Button,
+    Checkbox,
+    Divider,
+    Flex,
+    FlexItem,
+    Stack,
+    StackItem,
+    Switch,
+    Tooltip,
+} from '@patternfly/react-core';
 import { HelpIcon } from '@patternfly/react-icons';
 
 function DeploymentBaselines() {
     const [isAlertingOnViolations, setIsAlertingOnViolations] = React.useState<boolean>(false);
+    const [isExcludingPortsAndProtocols, setIsExcludingPortsAndProtocols] =
+        React.useState<boolean>(false);
 
     const handleAlertingOnViolations = (checked: boolean) => {
         setIsAlertingOnViolations(checked);
     };
 
+    const handleExcludingPortsAndProtocols = (checked: boolean) => {
+        setIsExcludingPortsAndProtocols(checked);
+    };
```

Comment:
Do we need the wrapper function, or could we just pass `setIsExcludingPortsAndProtocols` as the handler for the Checkbox component?

Reply:
Oh yes, you're right. I guess this isn't really necessary